lib/matplotlib/colorbar.py

```diff
@@ -1117,9 +1117,10 @@ def make_axes(parents, location=None, orientation=None, fraction=0.15,
     parent_anchor = kw.pop('panchor', loc_settings['panchor'])
     pad = kw.pop('pad', loc_settings['pad'])
 
-    # turn parents into a list if it is not already
-    if not isinstance(parents, (list, tuple)):
-        parents = [parents]
+    # turn parents into a list if it is not already. We do this w/ np
+    # because `ax=plt.subplots(1,1)` is an ndarray and is natural to
```

Comment:
Picky thing, but this comment is not quite right, should should be `fig, ax_arr = plt.subplots(squeeze=False)` or something like that.